tests/providers/amazon/aws/hooks/test_sagemaker.py

```diff
@@ -241,7 +241,7 @@
 }
 
 
-class TestSageMakerHook(unittest.TestCase):
```

Comment:
`unittest.TestCase` is incompatible with `@pytest.mark.parametrize` which I'm using below.
Given the fact that this inheritance was unnecessary, I simply removed it.

Reply:
Replace `unittest` classes by native class always welcome.